src/algorithms/kss.jl

```diff
@@ -34,7 +34,7 @@ end
 # Main function
 
 """
-    kss(X::AbstractMatrix{<:Real}, d::AbstractVector{<:Integer};
+    kss(X::AbstractMatrix{T} where T<:Number, d::AbstractVector{<:Integer};
```

Comment:
Let's make the style consistent here:

```suggestion
    kss(X::AbstractMatrix{<:Number}, d::AbstractVector{<:Integer};
```